### DISCUSSION TITLE
Floating labels and validation feedback messages

### DISCUSSION THREAD

**Discussion by mabasic** [twbs/bootstrap#33405] [Q&A] [answered]
I am working with Bootstrap 5 beta 2, and my issues is that when I set `is-invalid` on input which is in `form-floating` and use a `invalid-feedback` element it overlaps with input value and label.

![Screenshot (5)](https://user-images.githubusercontent.com/1839930/111769126-1d752880-88a9-11eb-839e-1dabdced7d60.png)

Is this a known issues or am I missing something?

**Comment by ffoodd** [comment]
Did you find any related issue? I don't remember any so I don't think this is known: I'd recommend you move this to an issue—and please provide a reduced test case if you can 👌

**Comment by mabasic** [comment, marked as answer]
My bad, it would seem that for floating labels when `is-invalid` class is applied to the input element, the error should be set in the label, not in `.invalid-feedback` element.

https://getbootstrap.com/docs/5.2/forms/floating-labels/#example
